jsapp/js/components/common/koboDropdown.tsx

```diff
@@ -1,27 +1,52 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import autoBind from 'react-autobind';
 import bem, {makeBem} from 'js/bem';
 import {
-  createEnum,
   KEY_CODES,
 } from 'js/constants';
 import koboDropdownActions from './koboDropdownActions';
 import './koboDropdown.scss';
 
-export const KOBO_DROPDOWN_THEMES = createEnum([
-  'light',
-  'dark',
-]);
+export enum KOBO_DROPDOWN_THEMES {
```

Comment:
I am wondering how should we name the `enum`s we create in TS. For now the `const`s we had followed the `SOME_NAME` idea, so it's clear that we have constant and enum-ish thing. But with TS we can clearly see what the thing is, as we see the definition whenever we use it. What do you think about just using `SomeName`?

Reply:
I also initially went with naming the enums as singular not plural, but I think we should keep going with plural 😅

Reply:
I was kind of confused by this too. I was thinking of just using `SomeName` for `enums` but it looks like syntax that should be used for types, so it felt a bit odd accessing attributes like `KoboDropdownThemes.light` versus `KOBO_DROPDOWN_THEMES.light`

Although, with what you said in the above comment:
> Enums in TypeScript are both a data to be used in code and a type

makes me feel like we should go with that after all, as it can double as a type and accessed

Reply:
Actually if we're following `textBox.tsx` as an example this is already the case :P